Src/FluentAssertions/Licensing.cs

```diff
@@ -0,0 +1,12 @@
+namespace FluentAssertions;
+
+/// <summary>
+/// Provides access to the licensing state of Fluent Assertions
+/// </summary>
+public static class Licensing
+{
+    /// <summary>
+    /// Allows to disable or enable the warning that reminds consumers about the license requirements for commercial use.
+    /// </summary>
+    public static bool AcknowledgeSoftWarning { get; set; } = true;
```

Comment:
As an end user I'm not sure I would know what "soft warning" refers to.

I find "Acknowledge" too vague as it means "accept or admit the existence or truth of something".

What if one could call `License.Accept = true`?

Reply:
But accept what?

Reply:
> But accept what?

The license.

Reply:
Alright. I can live with that.